lib/animina_web/components/layouts/app.html.heex

```diff
@@ -4,13 +4,11 @@
 
 <.desktop_top_text />
 
-<div class="w-[100%] pt-4  flex justify-center">
+<div class="w-[100%] pt-4  flex justify-start items-start">
```

Comment:
This should be in a separated PR. It is not relevant for #177

Reply:
This was from merging main into my branch.